packages/orchestrator-ui-components/src/theme/defaultOrchestratorTheme.ts

```diff
@@ -40,7 +40,7 @@ export const defaultOrchestratorTheme: EuiThemeModifications = {
             mediumShade: '#535966',
             darkShade: '#98A2B3',
             darkestShade: '#D4DAE5',
-            fullShade: '#FFFFFF',
+            fullShade: '#000000',
```

Comment:
Are we sure about this? It's the total opposite

Reply:
Yeah, I though it to be a quickfix, but it only fixes some places (like engine status badge), but make it worse at other places. I will revert it, today I will speak with Vishal on the dark theme design.
<img width="775" alt="image" src="https://github.com/workfloworchestrator/orchestrator-ui-library/assets/83226590/66ec4c52-1923-4c16-8892-1811105a836e">
dev:
<img width="778" alt="image" src="https://github.com/workfloworchestrator/orchestrator-ui-library/assets/83226590/241977c5-b13f-4ee3-b0bb-44cae61fd2d4">

Reply:
yes, I will revert it. It fixes the engine badge text, but it probably more an issue of the component than the theme. I have a call today with Vishal to discuss the dark theme design.